woocommerce/woocommerce-paypal-payments
# AXO: Add a Continue button next to the email input (3208)

### Description

This PR adds a Continue button near the email input to better communicate the action required to trigger Fastlane email lookup.

It also adds a description to the Fastlane payment gateway radio selection.


### Steps to Test





1. Open the checkout as a guest to enter the Fastlane checkout experience.
2. Notice the button next to the email input.
3. On mobile make sure the button displays below the watermark.
4. Make sure the overlay and spinner display correctly during email lookup.
5. Make sure the button is hidden when a different payment gateway gets selected.
6. Verify the payment gateway description displays correctly next to the radio.


### Screenshots

#### Desktop
|Before|After|
|-|-|
|![before_desktop](https://github.com/woocommerce/woocommerce-paypal-payments/assets/905781/c9ed3531-019d-41f6-be9d-f9a94601de5a)|![Checkout_–_WooCommerce_PayPal_Payments](https://github.com/woocommerce/woocommerce-paypal-payments/assets/905781/0ce9937e-0689-4af3-9994-d133fefaca8e)|

#### Mobile
|Before|After|
|-|-|
|![mobile_before](https://github.com/woocommerce/woocommerce-paypal-payments/assets/905781/d027cb55-80c0-4183-8e8f-b5a7d5cd299f)|![Checkout_–_WooCommerce_PayPal_Payments-2](https://github.com/woocommerce/woocommerce-paypal-payments/assets/905781/c51edf03-ffd7-4094-856b-81f6f8752af3)|
